README.md

```diff
@@ -1,10 +1,11 @@
 # rspec-autotest [![Build Status](https://secure.travis-ci.org/rspec/rspec-autotest.svg?branch=master)](http://travis-ci.org/rspec/rspec-autotest) [![Code Climate](https://codeclimate.com/github/rspec/rspec-autotest.svg)](https://codeclimate.com/github/rspec/rspec-autotest) [![Coverage Status](https://coveralls.io/repos/rspec/rspec-autotest/badge.png?branch=master)](https://coveralls.io/r/rspec/rspec-autotest?branch=master)
 
-rspec-autotest provides integration between autotest and RSpec
+rspec-autotest provides integration between autotest and RSpec, autotest is not included
```

Comment:
nit: I'd use a full stop rather than a comma here.

LGTM otherwise though (assuming build failures can't possibly be related)